Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -249,22 +642,46 @@ The following rules need to be followed, in addition to the guidelines described
    Make sure sequences are tested for all modes of operation, and as applicable, for winter, shoulder
    and summer days.
 
+#. Perform code text and, if applicable, figure inspection to:
+      - ensure there are no remaining typos
+      - ensure no element `public` was added automatically after the `protected` code section
 
+#. Use full paths to blocks, that is paths that start with `Buildings.Controls...`
 
-.. _sec_val:
+#. Final steps for a pull requests:
+      - make sure that the latest base branch is merged into the development branch
+      - after creating a pull request observe that the files changed pertain to the 
+        issue being addressed with the development branch
 
-Validation
-----------
+#. For general rules on validation models see :numref:`sec_val`. Specifically for the 
+   sequence development ensure that:
+      - the instances of models being tested are listed at the top of the code text
```

Comment:
I don't understand this line.

Reply:
If I am testing a MZ VAV controller then an instance of it should come before any other block instances included in the test.

Reply:
It is our own recommendation, similarly to ordering of types or instances.

Reply:
I rephrased, see if it works